src/pages/Vaults/Vault/VaultDashboard.tsx

```diff
@@ -31,11 +32,18 @@ const VaultDashboard = (): JSX.Element => {
 
   const vaultData = useGetVaultData({ address: selectedVaultAccountAddress });
   const transactions = useGetVaultTransactions(selectedVaultAccountAddress, vaultCollateral, bridgeLoaded);
+  const {
+    isIdle: identitiesIdle,
+    isLoading: identitiesLoading,
+    data: identities
+  } = useGetIdentities(bridgeLoaded);
+
   const { getCurrencyFromTicker, isSuccess: currenciesSuccess } = useGetCurrencies(bridgeLoaded);
 
   const vault = vaultData?.vaults?.find((vault: any) => vault.collateralId === vaultCollateral);
+  const displayName = identities?.get(selectedVaultAccountAddress);
```

Comment:
hi @nud3l this isn't critical, so it could be a TODO but I'd prefer that we fetched the identities from the `get-vault-data` and `use-get-vault-data` utilities `vault.displayName` and append it to the vault data, rather than fetching it separately. i.e. we end up with `vault.displayName`.

Reply:
I did that on purpose since identities can also be assigned to other user groups. So eventually, you will have this for collators, lenders, LPs, and even regular users.

Reply:
Ah nice, thanks :) yes definitely makes sense to have it as a separate hook, but what I was getting at was calling the identities hook from get-vault-data so that it can be retuned alongside the rest of the vault data. It's very minor though, and based on what you've said it might be better to call it separately anyway.